extensions/amp-jwplayer/amp-jwplayer.md

```diff
@@ -69,27 +69,58 @@ Example:
 </amp-jwplayer>
 ```
 
+Contextual Article Matching
+
+Example:
+
+```html
+  <amp-jwplayer
+      data-player-id="BjcwyK37"
+      data-media-id="BZ6tc0gy"
+      data-content-search="__CONTEXTUAL__"
```

Comment:
Is `__CONTEXTUAL__` a common name in the JwPlayer domain? I'm worrying that we're setting magical strings in something that would be better defined as an attribute or a combination of them.

Reply:
@alanorozco  Yes, this is common verbiage for JW Player. Understand the concern, but a user can also set this to any string value they desire, so we'd prefer this to stay a string.

Reply:
Since it's domain-specific, this is fine. 

This LGTM technically, will add one more person for documentation review.